scripts/darwin.sh

```diff
@@ -62,5 +66,23 @@ for disk in $DISKS; do
     echo "system: False"
   fi
 
+  partitions="`diskutil list $disk | get_device_identifiers`"
+  partitionmountpoints=""
+
+  for partition in $partitions; do 
+    partitioninfo="`diskutil info $partition`"
+
+    partitionmountpoint=`echo "$partitioninfo" | get_key "Mount Point"`
+
+    if [ "$partitionmountpoint" ]; then
+      partitionmountpoints=$partitionmountpoints,$partitionmountpoint
+    fi
+  done
+  
+  # trim leading ,
+  partitionmountpoints=${partitionmountpoints#,}
```

Comment:
If we move the array parsing code to the JS side of this module, it means that we can also take care of trimming there, which I believe is more convenient.